lib/inets/doc/src/httpc.xml

```diff
@@ -66,6 +66,7 @@
     <marker id="DATA_TYPES"></marker>
     <p>Type definitions that are used more than once in
       this module:</p>
+    <p><c>autoredirectoptions() = true | false | force</c></p>
```

Comment:
If we go for my suggestion we do not need the "force" and this new description. If we for some reason should decide we want this description it could use the _ for readability.